src/fabric/src/fabric_view.erl

```diff
@@ -469,8 +481,14 @@ fix_skip_and_limit(#mrargs{} = Args) ->
     {CoordArgs, WorkerArgs} =
         case couch_mrview_util:get_extra(Args, partition) of
             undefined ->
-                #mrargs{skip = Skip, limit = Limit} = Args,
-                {Args, Args#mrargs{skip = 0, limit = Skip + Limit}};
+                #mrargs{skip = Skip, limit = Limit, max_view = Max} = Args,
```

Comment:
The `max_view` field here I think is always 268435456? In that case adding or not adding an offset to it probably doesn't make a large difference, as the value is already huge. But this also point to a problematic spot. When previously we limited our per-node emits when user passed in a limit, we won't be doing that any longer and start emitting all the rows. I _think_ it should be ok given that we the rexi streaming replies (acks) and cleanup happens properly. But if it doesn't then this could add significant processing load on the workers.